spec/index.bs

```diff
@@ -13999,6 +13999,18 @@ interface GPUCanvasContext {
                 1. Let |descriptor| be the
                     [$GPUTextureDescriptor for the canvas and configuration$](|this|.{{GPUCanvasContext/canvas}}, |configuration|).
                 1. Set |this|.{{GPUCanvasContext/[[configuration]]}} to |configuration|.
+
+                    <div class=note heading>
+                        This spec requires supporting HDR via the {{GPUCanvasConfiguration/toneMapping}} option.
+                        If a user agent exposes HDR capabilities (i.e. exposes <l>'@media/dynamic-range'</l> `high`)
+                        but (nonconformantly) does *not* support HDR WebGPU canvases, then the
+                        {{GPUCanvasConfiguration/toneMapping}} member should *not* exist in
```

Comment:
@greggman [suggests](https://github.com/webgpu/webgpu-samples/pull/455#issuecomment-2425427326) (IIUC) that instead of `toneMapping` being missing, `toneMapping` should always be there but `toneMapping.mode` may be missing.

Reply:
There were no strong feelings about this in the meeting so we'll leave this as is. If `toneMapping` is exposed then `mode` should be exposed.